tiledb/sm/array_schema/dimension.cc

```diff
@@ -1613,25 +1619,29 @@ Status Dimension::check_tile_extent() const {
   if (!is_int) {
     // Check if tile extent is negative or 0
     if (*tile_extent <= 0)
-      return LOG_STATUS(Status_DimensionError(
-          "Tile extent check failed; Tile extent must be greater than 0"));
+      throw DimensionException(
+          name() + " [check_tile_extent] Tile extent must be greater than 0");
```

Comment:
Same comment as above (drop `[check_tile_name]` and make it more explicit what `name()` refers to).